grpclb/grpc_lb_v1/messages/messages.proto

```diff
@@ -15,7 +15,7 @@
 syntax = "proto3";
 
 package grpc.lb.v1;
-option go_package = "messages";
+option go_package = "google.golang.org/grpc/grpclb/grpc_lb_v1/messages";
```

Comment:
Changing this option also changes the output file path. (Run `make proto` to see the result).

It seems our current proto structure (or just the way we run `protoc`) doesn't work well with this new `go_package`. We need to fix that.

Reply:
The fix for this is to change the `--go_out` parameter in `grpclb_test.go`
```diff
diff --git a/grpclb/grpclb_test.go b/grpclb/grpclb_test.go
index 46c1fe5..70e390e 100644
--- a/grpclb/grpclb_test.go
+++ b/grpclb/grpclb_test.go
@@ -16,8 +16,8 @@
  *
  */
 
-//go:generate protoc --go_out=plugins=:. grpc_lb_v1/messages/messages.proto
-//go:generate protoc --go_out=Mgrpc_lb_v1/messages/messages.proto=google.golang.org/grpc/grpclb/grpc_lb_v1/messages,plugins=grpc:. grpc_lb_v1/service/service.proto
+//go:generate protoc --go_out=plugins=:$GOPATH2 grpc_lb_v1/messages/messages.proto
+//go:generate protoc --go_out=plugins=grpc:$GOPATH grpc_lb_v1/service/service.proto
 
 // Package grpclb_test is currently used only for grpclb testing.
 package grpclb_test

```

Reply:
Nice, thanks!